client/shaders/nodes_shader/opengl_fragment.glsl

```diff
@@ -27,7 +47,7 @@ float intensity (vec3 color)
 
 float get_rgb_height (vec2 uv)
 {
-	return intensity(texture2D(baseTexture,uv).rgb);
+	return intensity(texture2D(baseTexture, clamp(uv, 0.0, 0.999)).rgb);
```

Comment:
Why do this?  Accessing pixels outside the range (0..1) should just wrap around, no?